compiler/rustc_trait_selection/src/solve/assembly/mod.rs

```diff
@@ -455,7 +457,7 @@ impl<'tcx> EvalCtxt<'_, InferCtxt<'tcx>> {
         goal: Goal<'tcx, G>,
         candidates: &mut Vec<Candidate<'tcx>>,
     ) {
-        let tcx = self.tcx();
+        let tcx = self.interner();
```

Comment:
🤔 this feels a bit meh, have some thoughts but they aren't blocking so let's chat about this the next time we meet in sync